lib/charms/layer/openstack.py

```diff
@@ -140,10 +140,15 @@ def manage_loadbalancer(app_name, members):
 def cleanup():
     # note: we don't bother cleaning up the SG because it's a singleton
     # and can be reused in other / future deployments
-    for lb in LoadBalancer.get_all():
+    lbmanager = LoadBalancer('cleanup', None, None, None, None, None)
+    for lb in lbmanager.get_all():
         try:
-            lb.delete()
+            log("Deleting loadbalancer '{}'".format(lb['name']))
+            _openstack('loadbalancer', 'delete', '--cascade', lb['name'],
```

Comment:
Does this work for Neutron-based LBs?

Reply:
Do we even support using neutron lbaas with k8s? Its deprecated in Openstack and replaced by Octavia. In any case, right now the charm does nothing so better to have it work for Octavia that neither?